README.md

```diff
@@ -53,9 +53,38 @@ Name | Description
 In this simple example, we're going show you how to setup an endpoint where we can get a list of `dogs` as well as
  create and update `dogs`.
 
+ ### Create the Model
```

Comment:
The space at the beginning of this line is throwing-off the markdown formatting (it doesn't appear as a header).